### ISSUE TITLE
bit export <collection-name> <component-id> shows `fork` message

### ISSUE THREAD

**Issue by odedre** [teambit/bit#2159]
### Describe the bug

imported my component , made modifications and tried to export back to the same collection with the following command `bit export bitsrc.billing configs/bluesnap-plans --eject`.
I got the following message:

bit is about to fork the following components and export them to bitsrc.billing.
	configs/bluesnap-plans
also, if they're staged, bit will not change their status to exported unless '--set-current-scope' flag is used.

there are additional flags for the 'export' command to specifically handle forking components:
1. '--include-dependencies' exports all dependencies to the destination alongside the component.
2. '--set-current-scope' sets your workspace to use the destination scope as the main remote for the component.
3. '--rewire' changes all dependencies to point to the new destination.

### Expected Behavior

the component should not be forked to a different collection I am exporting to the original collection.

### Screenshots, exceptions and logs
![Screen Shot 2019-11-25 at 1 11 13 PM](https://user-images.githubusercontent.com/15064762/69535674-18d68380-0f85-11ea-9d7f-058a0afe5757.png)

### Specifications

- Bit version: 14.4.3

**Comment by KutnerUri** [comment]
I also got this warning.
It seems bit export does not remember remote scope when **using local name**.

![Screen Shot 2019-12-05 at 17 31 14](https://user-images.githubusercontent.com/5400361/70251217-c3ba1f00-1787-11ea-8263-c5b36da3d184.png)

using the full name works.
```
bit export bit.javascript bit.javascript/template/react-native
```
but seems a little redundant.

**Bonus:**
can you add the original scope?
```
bit is about to fork...
    template/react-native (was template/react-native)
```
(or in any other template)

**Comment by GiladShoham** [comment]
@KutnerUri Thanks for reporting this, we will address it soon.

**Comment by davidfirst** [comment]
I found the issue. 
This is happening when specifying an id without scope-name. 
Due to a bug with `scope.getParsedId`, the id was parsed incorrectly. e.g. for a name  "bar/foo", it was parsed as: `{ scope: "bar", name: "foo" }`, instead of `{ scope: null, name: "bar/foo" }`. When trying to export to a different scope than the original one, Bit prompts with the fork message.
This has been fixed in this PR: https://github.com/teambit/bit/pull/2591.
